manifests/install.pp

```diff
@@ -213,6 +213,9 @@
       if $::python::version =~ /^3/ {
         $pip_category = undef
         $pip_package = 'python3-pip'
+      } elsif ($::osfamily == 'RedHat') and (versioncmp($::operatingsystemmajrelease, '7') >= 0) and ($python::use_epel == true) {
```

Comment:
I probably misinterpreted `$use_epel` when writing this. I'll remove it.

As far as #349 is concerned: it doesn't do any spec tests and uses a regex on a fact that can be done cheaper and more effectively with `versioncmp()` and `$::operatingsystemmajrelease`.